lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1710,6 +1710,47 @@ static LogicalResult verifySubfieldOp(SubfieldOp op) {
   return success();
 }
 
+/// Check that a value is driven by a constant value. Looks through subindex and
+/// subfield ops.
+static bool isDrivenByConstant(Value op) {
+  for (auto &use : op.getUses()) {
```

Comment:
LLVM style prefers not using recursion to walk operations.  (and in isConstant).

Reply:
+1

Reply:
Will be fixed in the rebase.